Update nouns.mdx (#39)
medzuslovjansky/medzuslovjansky.github.io: docs/learn/grammar/nouns.mdx

> - After a soft consonant, the endings `-om`, `-ov` and `-y` are changed to **-`em`, -`ev`, -`e`**.

Reviewer: ```suggestion
- After a soft consonant, the endings `-om`, `-ov` and `-y` are changed to -`em`, -`ev`, -`e`.
```

> - Animate nouns have the nominative plural ending `-i` instead of the ending **-`y`/-`e`** of inanimate nouns.

Reviewer: ```suggestion
- Animate nouns have the nominative plural ending `-i` instead of the ending `-y`/`-e` of inanimate nouns.
```

- Neuter nouns do not have a vocative separate from the nominative.
- In the genitive plural, neuter nouns of the first declension have a zero ending `-Ø`. In cases when this leads to impossible consonant clusters, an `-e-` is inserted before `-j` or after a soft consonant, or an `-o-` between hard consonants: _`okn-o` > :isv[ok**o**n]_, _`morj-e` > :isv[mor**e**j]_.

> There is also special group of neuter nouns with the ending `-e` (in scientific orthography: `-ę`), for example `ime` „name” (stem: `imen-`) and `tele` „calf” (stem: `telęt-`). It also includes a few words on `-o`, for example `nebo` „heaven” (stem: `nebes-`). In Old Church Slavonic they belonged to a special declension, which nowadays has vanished in most languages. They can be inflected as ordinary neuter nouns (as if their nominatives were :isv[imeno] and :isv[teleto]), they can also be declined according to the more archaic [athematic declension][1].

Reviewer: ```suggestion
There is also special group of neuter nouns with the ending `-e` (in scientific orthography: `-ę`), for example :isv[ime] „name” (stem: `imen-`) and :isv[tele] „calf” (stem: `telęt-`). It also includes a few words on `-o`, for example :isv[nebo] „heaven” (stem: `nebes-`). In Old Church Slavonic they belonged to a special declension, which nowadays has vanished in most languages. They can be inflected as ordinary neuter nouns (as if their nominatives were :isv[imeno] and :isv[teleto]), they can also be declined according to the more archaic [athematic declension][1].
```

This declension type includes nouns of all three genders, but most numerous among them are neuter nouns. The following subtypes can be distinguished:

- masculine nouns on `-enj`, a tiny group including e.g. :isv[kamenj] „stone” and :isv[korenj] „root”
- neuter nouns on -`me`/-`men`-, containing numerous nouns like :isv[ime] „name”, :isv[rame] „shoulder”, :isv[brěme] „burden”
- neuter nouns on -`e`/-`et`-, referring to children and young animals, e.g. :isv[tele] „calf”, :isv[prase] „piglet”
- neuter nouns on -`o`/-`es`-, a group of only a few words, e.g. :isv[nebo] „heaven”
- feminine nouns on `-ov`, e.g. :isv[crkov] „church”, :isv[mrkov] „carrot”
- feminine nouns on **-`i`/-`er`-**, containing only two nouns: :isv[mati] „mother” and :isv[doči] „daughter”

<AthematicDeclension />

The entire athematic declension can be avoided by inflecting these words according to the regular declensions, as most other Slavic languages do. In that case:

- :isv[kamenj] is inflected like an ordinary masculine noun (:abbr[gen.] :isv[kamenja], :abbr[dat.] :isv[kamenju])

> - :isv[ime] and `tele` are inflected like :isv[slovo], as if their nominatives were :isv[imeno] and :isv[teleto]: :abbr[gen.] :isv[imena], :isv[teleta], :abbr[dat.] :isv[imenu], :isv[teletu]

Reviewer: ```suggestion
- :isv[ime] and :isv[tele] are inflected like :isv[slovo], as if their nominatives were :isv[imeno] and :isv[teleto]: :abbr[gen.] :isv[imena], :isv[teleta], :abbr[dat.] :isv[imenu], :isv[teletu]
```

- :isv[nebo] is inflected like :isv[slovo] too (:abbr[gen.] :isv[neba], :abbr[dat.] :isv[nebu])

> - :isv[crkov] is inflected like :isv[kost] (:abbr[gen.] :isv[crkvi]); instead of `crkov`, it is also possible to use the form :isv[crkva], which is inflected lke :isv[žena] (:abbr[gen.] :isv[crkvy], :abbr[dat.] :isv[crkvě])

Reviewer: ```suggestion
- :isv[crkov] is inflected like :isv[kost] (:abbr[gen.] :isv[crkvi]); instead of :isv[crkov], it is also possible to use the form :isv[crkva], which is inflected lke :isv[žena] (:abbr[gen.] :isv[crkvy], :abbr[dat.] :isv[crkvě])
```

> - :isv[mati] is inflected like :isv[kost]: :abbr[gen./dat]. :isv[materi]

Reviewer: ```suggestion
- :isv[mati] is inflected like :isv[kost]: :abbr[gen.]/:abbr[dat.] :isv[materi]
```

## Irregular nouns \{#irregular-nouns}

Interslavic is kept as regular as possible, but a few cases of irregularity cannot be avoided without defying naturalism. The following nouns have an irregular plural (all four inflected like a feminine noun of the `kost` type):

- :isv[člověk] (:abbr[m.]) „human being, person”, :abbr[pl.] :isv[ljudi] „people”
- :isv[děte] (:abbr[gen.] :isv[děteta] or :isv[dětete]) (:abbr[n.]) „child”, :abbr[pl.] :isv[děti] „children”
- :isv[oko] (:abbr[n.)] „eye”, :abbr[pl.] :isv[oči] „eyes”
- :isv[uho] (:abbr[n.)] „ear”, :abbr[pl.] :isv[uši] „ears”

Regular plurals (:abbr[člověki], :abbr[oka] etc.) can be used as well, but some of them sound very strange to the Slavic ear, even though they will be understood anyway.

## Indeclinable nouns \{#indeclinable-nouns}

> Borrowed international vocabulary ending in `-e`, `-i` or `-u` (e.g. :abbr[alibi], :abbr[hobi], :abbr[intervju], :abbr[kafe], :abbr[kakao], :abbr[kliše], :abbr[menju], :abbr[tabu], :abbr[taksi]) and abbreviations are indeclinable. It is not impossible to add case endings anyway, but in that case it is best to separate them from the noun with an apostrophe: :abbr[togo alibi’a], :abbr[te intervju’y], :abbr[tyh taksi’ov] itd.

Reviewer: ```suggestion
Borrowed international vocabulary ending in `-e`, `-i` or `-u` (e.g. :isv[alibi], :isv[hobi], :isv[intervju], :isv[kafe], :isv[kakao], :isv[kliše], :isv[menju], :isv[tabu], :isv[taksi]) and abbreviations are indeclinable. It is not impossible to add case endings anyway, but in that case, it is best to separate them from the noun with an apostrophe: :isv[togo alibi’a], :isv[te intervju’y], :isv[tyh taksi’ov], etc.
```
